skops/io/_general.py

```diff
@@ -113,17 +160,37 @@ def function_get_state(obj: Any, save_context: SaveContext) -> dict[str, Any]:
     return res
 
 
-def function_get_instance(state, load_context: LoadContext):
-    loaded = _import_obj(state["content"]["module_path"], state["content"]["function"])
-    return loaded
+class FunctionNode(Node):
+    def __init__(self, state, load_context: LoadContext, trusted=False):
+        super().__init__(state, load_context, trusted)
+        # TODO: what do we trust?
+        self.trusted = self._get_trusted(trusted, [])
+        self.children = {"content": FunctionType}
+        self.content = state["content"]
+
+    def _construct(self):
+        return _import_obj(self.content["module_path"], self.content["function"])
+
+    def _get_function_name(self):
+        return self.content["module_path"] + "." + self.content["function"]
+
+    @property
+    def is_safe(self):
+        return self._get_function_name() in self.trusted
+
+    def get_unsafe_set(self):
+        if self.is_safe:
+            return set()
```

Comment:
same here, once we pass `trusted` down or when we start trusting a few functions from numpy/scipy, this would be covered.